bundler/lib/bundler/man/bundle-init.1.ronn

```diff
@@ -16,6 +16,8 @@ created [`Gemfile(5)`][Gemfile(5)].
 
 * `--gemspec`:
   Use the specified .gemspec to create the [`Gemfile(5)`][Gemfile(5)]
+* `--gemfile`:
+  Use the specified gemfile instead of the [`Gemfile(5)`][Gemfile(5)]
```

Comment:
I'll clarify this help to make it explicit that the flag takes a explicit name like in https://github.com/rubygems/rubygems/pull/6046#discussion_r1085935335, but I'll do it quickly myself while giving this a final rebase.